app/helpers/api/ordapi/putFundingSource.js

```diff
@@ -3,7 +3,7 @@ import { logger } from '../../../logger';
 import { orderApiUrl } from '../../../config';
 
 const formatPutData = ({ fundingSource }) => ({
-  onlyGMS: fundingSource,
+  onlyGMS: fundingSource === 'true',
```

Comment:
Assume you don't need to worry about case here?

Reply:
We should be good here to not worry about the case